src/lib/utils/feature-flags.ts

```diff
@@ -0,0 +1,2 @@
+export const isLeverageSuiteEnabled = () =>
+  process.env.NEXT_PUBLIC_VERCEL_ENV !== 'index-app-prod'
```

Comment:
For convenience and accessibility put this into a config file in the /src root folder?